exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,9 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+    return ( sentence.search("code") >= 0)
```

Comment:
Awesome. I used : (sentence.includes("code")) which is just another way of doing the same thing. I will try to remember yours.